### DISCUSSION TITLE
Can I scrape user followers without chalange if they are more than 60k-100k.

### DISCUSSION THREAD

**Discussion by ghost** [davidwickerhf/instaclient#26] [Q&A]
I am needing library with witch I can scrape user followers list more than 100k or more it will be ok with instaclient?

**Comment by schaumju** [comment]
Were you able to figure out one? I am searching for the same thing

**Reply by ghost** [reply]
I go to another library is called insta-clent check out at the functions

**Reply by schaumju** [reply]
Could you give me the link to the repo by chance?

**Reply by ghost** [reply]
Just search for insta-client is the first repo I am for the app

**Reply by schaumju** [reply]
https://github.com/lucent-sea/InstaTech_Client this is the first one. is it this?

**Reply by ghost** [reply]
Only instaclient without -

**Reply by schaumju** [reply]
https://github.com/davidwickerhf/instaclient ?

**Reply by schaumju** [reply]
aka the one we are in?

**Reply by ghost** [reply]
Yea is this last one

**Reply by schaumju** [reply]
So with the rate limit you can only get 10k followers per account per hr it seems. Did you use multiple accounts or just wait for the rate limit to expire? Also how did you keep your accounts from receiving challenges without proxies?

**Reply by ghost** [reply]
Its scrape 10k at a time after this you need to wait 5 - 10 minutes and continue with a code you will see how its work

**Reply by schaumju** [reply]
Oh it's 5-10 minutes not an hr? So you do like an infinite loop with a sleep once you reach the rate limit?

**Reply by ghost** [reply]
You need to set to return you code from the function I can't see at the moment how exactly to do this when you receive the code after rate limit to paste it and its continue from where is stop

**Reply by schaumju** [reply]
Yea I did that already. How reliable is your 5-10 min time frame? I slept for 10 min and still hit rate limit. I thought it was 10k per hr but idk

**Reply by ghost** [reply]
When I leave it for 10 min is continuing

**Reply by schaumju** [reply]
while True:
            while keepGoing:
                followers, cursor, keepGoing = client.get_followers(user=account,
                                                                    count=750,
                                                                    use_api=True,
                                                                    deep_scrape=False,
                                                                    callback_frequency=50,
                                                                    end_cursor=cursor)

                i += len(followers)
                print(i)
                follower_list += followers
            print("Rate limit for ", login, " at ", datetime.datetime.now())
            time.sleep(600)
            
            That is what my code looks like. I added a 3rd return value to get_followers() that returns False if rate limit is reached. This look like yours?

**Reply by ghost** [reply]
I just now realized the question is for this library I will check my code and will send it to you I think I ask the question before I find the way to scrape big amount of followers

**Reply by schaumju** [reply]
That would be amazing if you could. Thank you!

**Reply by schaumju** [reply]
@mikegrep Do you have an idea when you'd be able to send the code? I'm still getting stuck with some things with this library. I appreciate all the help!
